mulighetsrommet-api/src/main/resources/db/migration/R__tiltaksgjennomforing_admin_view.sql

```diff
@@ -12,7 +12,6 @@ select tg.id::uuid,
        tg.arena_ansvarlig_enhet,
        tg.avslutningsstatus,
        tg.tilgjengelighet,
-       tg.estimert_ventetid,
```

Comment:
Jeg tipper du må legge til en `drop view` i toppen for at dette skal funke, siden du endrer på kolonne-rekkefølgen her. Har du evt. testet om dette funker lokalt ved å kjøre migrasjonen mot en database som allerede hadde forrige tilstand?

Reply:
Evt må det to PR's til. Først en PR som fjerner fra viewet, deretter en PR som fjerner `estimert_ventetid`. Vil tro `V96` ikke får kjørt i dev/prod fordi kolonnen benyttes i dette viewet

Reply:
Jeg kjørte opp backend lokalt uten at den klagde. Det vil vel være det samme?